website/api/controllers/webhooks/receive-from-customer-fleet-instance.js

```diff
@@ -41,6 +41,10 @@ module.exports = {
     unauthorized: {
       responseType: 'unauthorized',
       description: 'This webhook request could not be verified.',
+    },
+    badRequest: {
+      responseType: 'badRequest',
+      description: 'Invalid MDM migration request.'
     }
```

Comment:
See related comment below.  A `badRequest` from Fleet would indicate that the requestor had sent something invalid to Fleet.  But actually, a badRequest from workspace one's API indicates that Fleet's internal logic sent something invalid to Workspace One, which is orthogonal.